frontends/web/src/routes/buy/info.tsx

```diff
@@ -65,23 +66,90 @@ class BuyInfo extends Component<Props, State> {
+                                                    <th>{t('buy.info.disclaimer.payment.table.description')}</th>
+                                                </tr>
+                                            </thead>
+                                            <tbody>
+                                                <tr>
+                                                    <td>{t('buy.info.disclaimer.payment.table.1_method')}</td>
+                                                    <td class={style.nowrap}>1.9 %</td>
+                                                    <td>{t('buy.info.disclaimer.payment.table.1_description')}</td>
+                                                </tr>
+                                                <tr>
+                                                    <td>{t('buy.info.disclaimer.payment.table.2_method')}</td>
+                                                    <td class={style.nowrap}>5.9 %</td>
+                                                    <td>{t('buy.info.disclaimer.payment.table.2_description')}</td>
+                                                </tr>
+                                                <tr>
+                                                    <td>{t('buy.info.disclaimer.payment.table.3_method')}</td>
+                                                    <td class={style.nowrap}>5.9 %</td>
+                                                    <td>{t('buy.info.disclaimer.payment.table.3_description')}</td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+                                    </div>
+                                    <p>{t('buy.info.disclaimer.payment.footnote')}</p>
+                                    <h2 class={style.title}>
+                                        {t('buy.info.disclaimer.security.title')}
+                                    </h2>
+                                    <p>{t('buy.info.disclaimer.security.description', { name })}</p>
+                                    <p>
+                                        <A class={style.link} href="https://shiftcrypto.ch/bitbox02/threat-model/">
+                                            {t('buy.info.disclaimer.security.link')}
+                                        </A>
+                                    </p>
+                                    <h2 class={style.title}>
+                                        {t('buy.info.disclaimer.protection.title')}
+                                    </h2>
+                                    <p>{t('buy.info.disclaimer.protection.description')}</p>
+                                    <p>
+                                        <A class={style.link} href="https://support.moonpay.com/hc/en-gb/articles/360009279877-What-are-your-supported-countries-states-and-territories-">
```

Comment:
you need to whitelist `"^https://support.moonpay.com/"` so that this link can be opened:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/moonpay/backend/backend.go#L1130